From: Andrew Morton <akpm@linux-foundation.org>
Subject: crash-add-kunit-tests-for-crash_exclude_mem_range-fix
Date: Thu Sep  4 08:07:47 PM PDT 2025

export crash_exclude_mem_range() to modules, for kernel/crash_core_test.c

Cc: Baoquan He <bhe@redhat.com>
Cc: Borislav Betkov <bp@alien8.de>
Cc: Coiby Xu <coxu@redhat.com>
Cc: Dave Young <dyoung@redhat.com>
Cc: fuqiang wang <fuqiang.wang@easystack.cn>
Cc: "H. Peter Anvin" <hpa@zytor.com>
Cc: Ingo Molnar <mingo@redhat.com>
Cc: Thomas Gleinxer <tglx@linutronix.de>
Cc: Vivek Goyal <vgoyal@redhat.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 kernel/crash_core.c |    1 +
 1 file changed, 1 insertion(+)

--- a/kernel/crash_core.c~crash-add-kunit-tests-for-crash_exclude_mem_range-fix
+++ a/kernel/crash_core.c
@@ -338,6 +338,7 @@ int crash_exclude_mem_range(struct crash
 
 	return 0;
 }
+EXPORT_SYMBOL_GPL(crash_exclude_mem_range);
 
 ssize_t crash_get_memory_size(void)
 {
_
